### PR TITLE
Avoid persisting clusters when GET endpoints are called

### PR DESCRIPTION
**Pull request by aureamunoz**
Fix https://github.com/halkyonio/primaza-poc/issues/326

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
   verify-in-kubernetes:
     name: Verify in Kubernetes
     runs-on: ubuntu-latest
-    needs: build
+#    needs: build
```
Comment (Sgitario) [inline .github/workflows/pr.yml:24]: This change seems unrelated to this pull request.

File: app/src/main/java/io/halkyon/services/KubernetesClientService.java [hunk 2]
```diff
@@ -296,15 +296,15 @@ public KubernetesClient getClientForCluster(Cluster cluster) throws ClusterConne
...
-            errorCluster.persist();
-            throw new ClusterConnectException(errorCluster, ex);
+            // Cluster errorCluster = Cluster.findByName(cluster.name);
+            // errorCluster.status = ClusterStatus.ERROR;
+            // if (ex.getCause() != null) {
+            // errorCluster.errorMessage = ex.getCause().getMessage();
+            // } else {
+            // errorCluster.errorMessage = ex.getMessage();
+            // }
+            // errorCluster.persist();
```
Comment (Sgitario) [inline app/src/main/java/io/halkyon/services/KubernetesClientService.java:306]: Is there any reason to keep these comments here?

**Review by Sgitario — COMMENTED** [review]
Apart from the two comments, the rest of the changes look good to me.

File: .github/workflows/pr.yml (hunk 1 above)
Comment (aureamunoz) [reply .github/workflows/pr.yml:24]: Good catch, exactly, this has nothing to do with this. I will revert

File: app/src/main/java/io/halkyon/services/KubernetesClientService.java (hunk 2 above)
Comment (aureamunoz) [reply app/src/main/java/io/halkyon/services/KubernetesClientService.java:306]: ah no! I will remove these lines. Thx

**Review by Sgitario — APPROVED** [review]

**Pull request merged**